src/commands/pwd_command.c

```diff
@@ -1,13 +1,29 @@
 #include <libft.h>
+#include <env/environment.h>
 #include <commands/commands.h>
 
+char	*get_pwd(char *buffer)
+{
+	const t_env	*pwd = find_variable(get_environment(), "PWD");
+
+	getcwd(buffer, BUFFER_SIZE);
+	if (!buffer && !pwd)
+		return (NULL);
+	else if (!buffer && pwd && pwd->value)
```

Comment:
`buffer` will never be `NULL`, it will always return a string. We have to check if `!(*buffer)` instead of `!buffer`

```C
if (!(*buffer) && !pwd)
	return (NULL);
else if (!(*buffer) && pwd && pwd->value)
	ft_strlcpy(&buffer[0], pwd->value, ft_strlen(pwd->value) + 1);
```